features/steps/givens/entities.py

```diff
@@ -58,4 +58,10 @@ def step_impl(context, inst, relationship_direction):
     attr_value = getattr(inst, attribute_name, NullAttribute)
     if attr_value is None:
         attr_value = NullAttribute
-    yield ValidationOutcome(instance_id = attr_value, severity = OutcomeSeverity.PASSED)
\ No newline at end of file
+    yield ValidationOutcome(instance_id = attr_value, severity = OutcomeSeverity.PASSED)
+
+
+@gherkin_ifc.step("All referenced instances")
+def step_impl(context, inst):
+    instances = context.model.traverse(inst)
```

Comment:
```suggestion
    # Note that this includes `inst` as the first element in this list
    instances = context.model.traverse(inst)
```

Just one word of caution